### ISSUE TITLE
Incompatible with 10.9.0

### ISSUE THREAD

**Issue by jet082** [firecore/InfuseSync#21]
As it says on the tin. The issue is the following:

```
[2023-10-12 00:55:41.734 +02:00] [ERR] Failed to load assembly "/var/lib/jellyfin/plugins/InfuseSync_1.4.2.0/InfuseSync.dll". This error occurs when a plugin references an incompatible version of one of the shared libraries. Disabling plugin
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Could not load file or assembly 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.

   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.Assembly.GetTypes()
   at Emby.Server.Implementations.Plugins.PluginManager.LoadAssemblies()+MoveNext()
System.IO.FileNotFoundException: Could not load file or assembly 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
```

I believe this is due to https://github.com/jellyfin/jellyfin/pull/10138 but I could be wrong.

**Comment by firecore** [comment]
We have just posted a [beta version of 1.5.0](https://github.com/firecore/InfuseSync/releases/tag/v1.5.0) which is available to test.

We are still doing some final internal testing but things are looking good.

**Comment by firecore** [comment]
We've released the [1.5.0 version of InfuseSync](https://github.com/firecore/InfuseSync/releases/tag/v1.5.0) which adds support for Jellyfin 10.9.

We're working with the Jellyfin admins to get this new version added to the main Jellyfin plugin catalog, but in the meantime you can install the update by following the [steps in this post on our forum](https://community.firecore.com/t/add-infusesync-plugin-support-for-jellyfin-10-9/49006/22).
